daml-lf/language/src/main/scala/com/digitalasset/daml/lf/language/LanguageVersion.scala

```diff
@@ -74,11 +74,11 @@ object LanguageVersion {
   // All the stable and preview versions
   // Equals `Stable` if no preview version is available
   val EarlyAccessVersions: VersionRange[LanguageVersion] =
-    StableVersions
+    StableVersions.copy(max = v1_15)
```

Comment:
We should not make 15 stable.

Reply:
We're defining Early Access in terms of Stable, adding 1.15 to the list of early access versions - we don't modify Stable here. Or does .copy also mutate the original VersionRange?

Reply:
my bad.